### DISCUSSION TITLE
Welcome to audible-cli Discussions!

### DISCUSSION THREAD

**Discussion by mkb79** [mkb79/audible-cli#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andrewjasons** [comment]
Is there a way to convert the downloaded aaxc file into mp3 or m4b

**Reply by mkb79** [reply]
Please download ffmpeg with aaxc patch applied. You can get it from [here](https://www.gyan.dev/ffmpeg/builds/) for example. Then you can use my [plugin script](https://github.com/mkb79/audible-cli/blob/master/plugin_cmds/cmd_remove-encryption.py) for audible-cli or use ffmpeg manually with audible_iv and audible_key from voucher file.

**Reply by andrewjasons** [reply]
hi thankyou for your response and im sorry for asking such a basic question but, how can I run your plugin script in the audible-cli

**Reply by andrewjasons** [reply]
i literally know nothing about coding but i want to learn about these things

**Reply by mkb79** [reply]
Simply copy the script to your [plugin dir](https://github.com/mkb79/audible-cli#plugin-folder).

[Here](https://github.com/mkb79/audible-cli#app-dir)you can find the location to your app dir.

**Reply by andrewjasons** [reply]
i cant find my plugin dir and my app dir

**Reply by mkb79** [reply]
Which OS you have?

**Reply by andrewjasons** [reply]
i use windows 10

**Reply by andrewjasons** [reply]
C:\Users\sonson\AppData\Local\Audible i cant find anything other than config.tmol and .json

**Reply by andrewjasons** [reply]
i've tried to figure this blindly for more than 3 hours & im super frustrated rn

**Reply by mkb79** [reply]
Add a dir named "plugins" inside C:\Users\sonson\AppData\Local\Audible. In this folder copy the script!

**Reply by andrewjasons** [reply]
and then what command do i have to type in?

**Reply by mkb79** [reply]
Look [here](https://github.com/mkb79/audible-cli#custom-commands). When the filename is `cmd_remove-encryption.py`, then the command is `audible remove-encryption`!

**Reply by andrewjasons** [reply]
omg, for some reason i created AUDIBLE_PLUGIN_DIR which if used correctly will work, but because i dont know how to use this properly, it ends up making everything complicated. I then deleted that environment variable and try again the command and it works! thankyou so much for all your response!

**Reply by mkb79** [reply]
No problem. Don't worry to ask me again!

**Comment by Sloppyedwards** [comment]
I installed audible-cli and successfully download an *.aax file to c:\temp\audible. Next I downloaded cmd_remove-encryption.py to C:\Users\slong\AppData\Local\Audible\plugins\ but when I try to use this plugin, I am getting the following:
```
c:\Temp\audible>audible remove-encryption

Warning: entry point could not be loaded. Contact its author for help.

Traceback (most recent call last):
  File "C:\Users\slong\AppData\Local\Programs\Python\Python310\lib\site-packages\audible_cli\plugins.py", line 40, in decorator
    mod = import_module(mod_name)
  File "C:\Users\slong\AppData\Local\Programs\Python\Python310\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\slong\AppData\Local\Audible\plugins\cmd_remove-encryption.py", line 107
    <title>audible-cli/cmd_remove-encryption.py at master · mkb79/audible-cli</title>
                                                          ^
SyntaxError: invalid character '·' (U+00B7)
```
Any suggestions?

**Reply by mkb79** [reply]
Please take a look on line 107 of the file `cmd_remove-encryption.py` in the plugin dir. There seams to be a not valid sign in this line. Maybe you have entered something?!

**Reply by Sloppyedwards** [reply]
Thanks for the quick feedback!  Upon opening the plugin, I find that I had done a "Save link as" from the link above, which got me the html page, not the actual file contents.  I have remedied that error, and the command runs now, although I'm missing ffmpeg, so I'll have to remedy that if I want to keep going.  However, having obtained the *.aax file using your Audible API and CLI, since then I was able to use Inaudible to get *.mp3 files to listen to in my car.  Thanks for making the API and CLI available!

**Reply by mkb79** [reply]
Great to hear that! Maybe you can take a look at [AAXtoMP3](https://krumpetpirate.github.io/AAXtoMP3/) which works with `audible-cli` to convert your audiobooks!
